barretenberg/cpp/src/barretenberg/transcript/transcript.hpp

```diff
@@ -350,6 +350,31 @@ template <typename TranscriptParams> class BaseTranscript {
         return call_get_challenges();
     }
 
+    /**
+     * @brief Adds an element to the transcript.
```

Comment:
Can you update this description to mention that this doesn't implicitly add to the proof?